<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wd_msg</class>
 <widget class="QWidget" name="wd_msg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>41</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>499</width>
    <height>41</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>499</width>
    <height>41</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lb_name">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>11</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <italic>false</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>群成员name</string>
   </property>
  </widget>
  <widget class="QLabel" name="te_msg">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>281</width>
     <height>42</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>281</width>
     <height>42</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>281</width>
     <height>41</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 170, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_mem">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>41</width>
     <height>41</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>41</width>
     <height>41</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>41</width>
     <height>41</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
